### ISSUE TITLE
[Software]: Slax

### ISSUE THREAD

**Issue by Nelias** [open-neuromorphic/open-neuromorphic.github.io#423]
### Software Name

Slax

### Short Description

SNN Framework that has: BPTT, RTRL, FPTT, OTTT, OSTL, OTPE built-in. It supports NIR.

Paper: https://iopscience.iop.org/article/10.1088/2634-4386/ada9a8

Its main maintainer actually works at Google but idk if it is related to that company but he did JAX related things at Google so it might be...

Authors of the paper work at Forward Edge-AI.

### Source Code URL (GitHub/GitLab)

https://github.com/Intelligent-Microsystems-Lab/slax

### PyPI Package Name (if applicable)

slax

### Programming Language

Python

### License

Apache-2.0 license

### Key Dependencies

JAX

### Field of Application

Machine Learning

### Capabilities

- [ ] Supports Neuromorphic Hardware
- [x] Supports NIR (Neuromorphic Intermediate Representation)

### Maintainer(s)

Intelligent Microsystems Lab

### Detailed Overview

Slax is a JAX library built on top of Flax. Its purpose is to provide many learning rules and gradient approximations, with a heavy focus on training spiking neural networks. Some of these algorithms, however, are also applicable to any recurrent network.
